colorizer/README.md

```diff
@@ -0,0 +1,38 @@
+# Colorizer
+Tool to migrate an existing Pxl installation to Chroma.
+
+## Requirments
+- Access to the S3 bucket backing the Pxl installation
+- A service token for Chroma (refer to the config option `service_tokens` in Chroma)
+
+## Installation
+There is no prebuild executable of colorizer. Thus, you need the Rust toolchain installed along with the protobuf compiler.
+```
+cargo install --git https://github.com/svsticky/chroma.git colorizer
+```
+
+## Usage
+>Warning: Take a backup of Chroma's current S3 and database state!
+
+Simple: 
+```bash
+./colorizer \
+  --pxl-metadadata-file <Name of the metadata file on S3> \
+  --chroma-api <URL to Chroma, e.g. https://chroma.svsticky.nl> \
+  --chroma-service-token <Your service token> \
+  --s3-bucket-name <The name of the Pxl bucket> \
+  --s3-region <The region of the Pxl bucket> \
+  --s3-endpoint-url <The URL of the Pxl bucket> \
+  --s3-access-key-id <The access key ID of the Pxl bucket> \
```

Comment:
`clap` has an `env` feature already, although I'm not sure if can declare a value that can ONLY be sourced from the environment.

Reply:
I'd normally agree, but since this is a one-off command, I wouldn't put too much effort in supporting environment variables. If `clap` already picks those up, it is easy enough to update the documentation a little bit of course.

Reply:
Good catch.